.gitignore

```diff
@@ -6,4 +6,5 @@ build/
 .DS_Store
 coverage/
 .vscode/
-.sass-cache/
\ No newline at end of file
+.sass-cache/
+scss/
```

Comment:
I just wanted to clarify that you've excluded this from version control because it is only copied during the builds for the npm package? Makes sense, but could you please also remove the current scss folder that exists, otherwise we might end up with an increasingly out of date copy of the scss, that may get confusing. Thanks.

Reply:
yes, exactly, added it to ignore because it is the build command output and can be confusing with the actual `src/scss`
removed the folder now